evm/vm/flavors/homestead/opcodes.py

```diff
@@ -26,7 +26,7 @@
 
 
 HOMESTEAD_OPCODES = {
-    **copy.deepcopy(FRONTIER_OPCODES),
+    **copy.deepcopy(FRONTIER_OPCODES),  # noqa: E999
```

Comment:
Just a note for later: the `e999` really indicates that python2 will be angry about this.  It would be prudent for us to replace these {**something, **something_else}` with a proper non-mutating `merge(dict_a, dict_b)` such as the one from `toolz.dicttoolz.merge`.